crates/node/src/lib.rs

```diff
@@ -148,7 +148,10 @@ impl Node {
         let mut router_builder = MessageRouterBuilder::default();
         let metadata_builder = MetadataBuilder::default();
         let metadata = metadata_builder.to_metadata();
-        let networking = Networking::new(metadata_builder.to_metadata());
+        let networking = Networking::new(
+            metadata_builder.to_metadata(),
+            config.networking.retry_policy.clone(), // TODO: is this a reasonable way to access config & pass it in?
```

Comment:
We may want to pass in the entire `NetworkingOptions` struct in the future, but for now this seems like a reasonable approach.

Reply:
If we don't want to change the networking retry policy at runtime, then this works. If we want to enable changing an option at runtime, then one needs to pass in a `Live<NetworkingOptions>` which one then queries at every connection attempt.

Reply:
We can start like this and change it once we see the need for changing the retry policy dynamically.